File: _Internal/Model_NorthwindIB_NH/Entities/EmployeeTerritory.cs
namespace Models.NorthwindIB.NH
{
    using System;
    using System.Collections.Generic;

    public partial class EmployeeTerritory
    {
        public virtual int ID { get; set; }
        public virtual int EmployeeID { get; set; }
        public virtual int TerritoryID { get; set; }
        public virtual int RowVersion { get; set; }

        public virtual Employee Employee { get; set; }
        public virtual Territory Territory { get; set; }
    }
}

File: Samples_Unpublished/Java/NorthBreeze/WEB-INF/src/northwind/model/EmployeeTerritory.java
package northwind.model;

public class EmployeeTerritory {
    private int id;
    private int employeeID;
    private int territoryID;
    private int rowVersion;

    private Employee employee;
    private Territory territory;
    
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public int getEmployeeID() {
		return employeeID;
	}
	public void setEmployeeID(int employeeID) {
		this.employeeID = employeeID;
	}
	public int getTerritoryID() {
		return territoryID;
	}
	public void setTerritoryID(int territoryID) {
		this.territoryID = territoryID;
	}
	public int getRowVersion() {
		return rowVersion;
	}
	public void setRowVersion(int rowVersion) {
		this.rowVersion = rowVersion;
	}
	public Employee getEmployee() {
		return employee;
	}
	public void setEmployee(Employee employee) {
		this.employee = employee;
	}
	public Territory getTerritory() {
		return territory;
	}
	public void setTerritory(Territory territory) {
		this.territory = territory;
	}

}
